class AppException(val exceptin: Throwable?): Exception();
